plugins/inputs/cadvisor/mesos/plugin.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
Should we add some form of README or something to indicate that this was lifted from cadvisor? I've seen that in other places. It could help correlate the licenses here with where this code originates from.